### PR TITLE
TypeDoc -  Generate the api documentation for the application.

### PR DESCRIPTION
**Pull request by natarajanmca11**

### CODE REVIEW

**Comment by ludohenin** [comment]
Awesome, will merge it soon.

**Comment by natarajanmca11** [comment]
Thank  you @ludohenin 
I am new to git. That's why late check-in. 
Please link this request id with #174 and make #174 as obsolete.

**Comment by ludohenin** [comment]
Just tried it but Isn't missing a kind of `serve.docs` ?

**Comment by natarajanmca11** [comment]
Yes.I Will do it.

**Comment by tarlepp** [comment]
Should the ```docs``` folder to be included to ```.gitignore``` ?

**Comment by natarajanmca11** [comment]
`docs` folder contains lots of auto generated files. so we no need to keep it in the repository. 
I would like keep this project is very lightweight. What would you say?

**Comment by NathanWalker** [comment]
@natarajanmca11 this is excellent thank you, lgtm. @mgechev merge?

**Comment by ludohenin** [comment]
@NathanWalker I'm not quite in fact.

**Comment by mgechev** [comment]
@natarajanmca11 looks great! Lets merge this PR, although we are planning some drastic changes in the entire seed project.

**Comment by NathanWalker** [comment]
@mgechev Let me know if I can assist on any of the ***drastic*** changes :+1:  Curious what those are.

**Comment by ludohenin** [comment]
That's actually quite cool to navigate into angular source

**Comment by natarajanmca11** [comment]
Thank you all. I am really enjoy with this seed project. Learned a lot from here.

File: app/components/about/about.ts [hunk 1]
```diff
@@ -8,11 +8,15 @@ import {NameList} from '../../services/name_list';
   directives: [CORE_DIRECTIVES]
```
Comment (thanhdiepuu) [inline app/components/about/about.ts]: ew

**Pull request merged**